bucketeer/src/main/java/jp/bucketeer/sdk/BucketeerImpl.kt

```diff
@@ -58,7 +58,6 @@ internal class BucketeerImpl constructor(
 
   init {
     try {
-      ProviderInstaller.installIfNeeded(application)
```

Comment:
Removing this for the same reason described here https://github.com/ca-dp/bucketeer-android-sdk/pull/14#discussion_r489312436